telegram/commands/report/sleep.go

```diff
@@ -0,0 +1,74 @@
+package report
+
+import (
+	"activity-tracker/shared"
+	"activity-tracker/storage"
+	"fmt"
+	"time"
+)
+
+var (
+	timeFormat   = "03:04 pm"
+	locationTime = "America/Bogota"
+
+	reportSleepMessage = `Pillá pues cómo son las vueltas precios@ %s 🍆
+
+	Esta semana te has acostado a estas horas bb:
```

Comment:
i think we can create a report  when we return the total hours of sleep by day, with this proposed report the tracks of power naps will be ignored

Reply:
Done, now command track use the content and increment time sleep